net/pdnsd/files/pdnsd.init

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh /etc/rc.common
+
+START=65
+NAME=pdnsd
+DESC="proxy DNS server"
+
+DAEMON=/usr/sbin/pdnsd
+PID_FILE=/var/run/$NAME.pid
+CACHEDIR=/tmp/pdnsd
+CACHE=$CACHEDIR/pdnsd.cache
+
+USER=nobody
+GROUP=nogroup
+
+start() {
+	echo -n "Starting $DESC: $NAME"
```

Comment:
can be replaced with logger -t 'pdnsd' "Starting" or something

Reply:
Fixed done